Tests/UnitTests/fss-search-results.component.spec.ts

```diff
@@ -175,11 +175,12 @@ describe('FssSearchResultsComponent', () => {
         let css = "[id='" + item.BatchID.value + "']";
         const home = fixture.debugElement.query(By.css(css)).nativeElement;
         if (item.allFilesZipSize) {
-          expect(home.querySelector("ukho-button").textContent).toEqual("Download all");
+          expect(home.querySelector("a").textContent).toEqual("Download all");
```

Comment:
Could you please add an expectation to prove the `batches` collection is not empty - to prove at least one iteration of the forEach loop will be executed

Reply:
Modified as per review comment